features/environment.py

```diff
@@ -104,6 +104,8 @@ def before_all(context: Context) -> None:
     then capture an image.  This image is then reused by each feature, reducing
     test execution time.
     """
+    userdata = context.config.userdata
+    context.container_to_run = userdata.get("container_to_run")
```

Comment:
To align with the reuse_image config option name let's call this reuse_container

Reply:
I'm also thinking that maybe this should be an environment config too insteead of using userdata here?

But it's worth discussion as I see this as a separate developer tool when we are writing behave tests which maybe should live outside the expected CI configuration for Behave which uses environment variables.

Reply:
I don't see a reason as this is a unique configuration, just for development tests. This way, I think it is simpler, don't need to add it to `UAClientBehaveConfig` class

Reply:
I think that comment above was related to keeping things in userdata. but not the rename?

Reply:
Let me change the name for reuse_container